presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoControlUtils.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.teradata.prestomanager.agent;
+
+import com.google.gson.Gson;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.ws.rs.core.Response;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static com.teradata.prestomanager.agent.AgentFileUtils.getFileProperty;
+import static com.teradata.prestomanager.agent.CommandExecutor.executeCommand;
+import static com.teradata.prestomanager.agent.PrestoControlUtils.StopType.GRACEFUL;
+import static com.teradata.prestomanager.common.RequestBuilder.newRequestBuilder;
+import static java.lang.String.format;
+import static javax.ws.rs.client.Entity.entity;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON_TYPE;
+import static javax.ws.rs.core.Response.Status.ACCEPTED;
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+import static javax.ws.rs.core.Response.Status.OK;
+
+public final class PrestoControlUtils
+{
+    private static final Logger LOGGER = LogManager.getLogger(PrestoControlUtils.class);
+    private static final String LAUNCHER_SCRIPT = "/usr/lib/presto/bin/launcher";
+    private static final Path PRESTO_CONFIG_FILE = Paths.get("/etc/presto/config.properties");
+    private static final Gson GSON = new Gson();
+
+    private PrestoControlUtils() {}
+
+    public static Response startUsingRpm()
+    {
+        try {
+            if (executeCommand("rpm", "-q", "presto-server-rpm") != 0) {
```

Comment:
The ```isInstalled``` function didn't prevent the duplication of code since I removed the  ```try-catch``` block with the function. So I thought it's better to inline the function